### PR TITLE
Emmanuel Steve details

### PR DESCRIPTION
**Pull request by talk2coded**
Hello ! I just added my details, I hope to I'm hired soon.

### CODE REVIEW

**Review by shrutikapoor08 — CHANGES_REQUESTED** [review]
Hey Emmanuel! Looks like formatting is off. Can you enter details within `||` and in the correct section of Nigeria.

**Comment by talk2coded** [comment]
I’ve updated it 
Sorry about that

File: README.MD [hunk 1]
```diff
@@ -29,6 +29,8 @@ Country:
 - [China](#China)
 - [Sri Lanka](#Sri-Lanka)
 - [Nigeria](#Nigeria)
+|  |  |  |  |  |  |
```
Comment (shrutikapoor08) [inline README.MD:32]: can you remove this line?

**Comment by talk2coded** [comment]
Done

**Review by shrutikapoor08 — APPROVED** [review]

**Pull request merged**